plugin/mutate/source/dextool/plugin/mutate/backend/database/schema.d

```diff
@@ -411,6 +412,30 @@ struct MutantTimeoutCtx {
     State state;
 }
 
+import dextool.plugin.mutate.backend.database.type : Rationale;
+
+@TableName(markedMutantTable)
+@TablePrimaryKey("mutationStatusId")
```

Comment:
should probably be `st_id` because the name is changed by the `ColumnName` UDA further down